specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14/keyvault.json

```diff
@@ -0,0 +1,1359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-14-preview",
```

Comment:
>2018-02-14-preview [](start = 16, length = 18)

Is this based on an already exisiting API version?

Reply:
yes this is based of 2016-10-01 with the addition of the networkAcls property on VaultProperties

Reply:
Then, could you please submit the PR like this...
- First commit, copy and paste swaggers from already existing version.
- Second commit, modify the pasted swaggers so they now reflect the new API.
This way it is a lot easier to review, I can just look at the diffs :)